server/index.js

```diff
@@ -69,7 +70,8 @@ app.use(braveAlerter.getRouter())
 vitals.setupVitals(braveAlerter)
 
 async function handleAlert(location, alertType) {
-  const alertTypeDisplayName = helpers.getAlertTypeDisplayName(alertType)
+  // const alertTypeDisplayName = helpers.getAlertTypeDisplayName(alertType)
+  const alertTypeDisplayName = localHelpers.getAlertTypeDisplayName(alertType)
```

Comment:
As we discussed earlier, you can do this change directly in `brave-alert-lib` to take an (optional) language parameter instead of creating a new `localHelpers` function.